### ISSUE TITLE
Dependencies with private Git URL can cause high CPU usage.

### ISSUE THREAD

**Issue by ajhsu** [vscode-contrib/vscode-versionlens#86]
## Description
Dependencies with public Git URL works fine, but once it encountered private packages will raise high CPU usage in my case.

A privaite [Git URL](https://docs.npmjs.com/files/package.json#git-urls-as-dependencies) for example, is like:
`"internal-package": "git+ssh://git@gitlab.internal-domain.com/internal-package.git"`

## Symptoms

1. The spinning "v" on the top-right corner of working space that never stops.
2. Mid-High CPU usage: It continuously consuming about 50-70% of CPU (MBPR Late 2013 w/ i5 2.6 G) usage as soon as I opened up my `package.json`; And it will back to normal immediately when I closed the `package.json` file.

3. Some of error messages show up in [VSCode's Developer Tools](https://code.visualstudio.com/docs/extensions/debugging-extensions#_troubleshooting-extensions):

```
[Extension Host] Error: NPM: parseNpmArguments TypeError: Cannot read property 'type' of undefined
  at /Users/ajhsu/.vscode/extensions/pflannery.vscode-versionlens-0.19.1/out/extension-bundle.js:1246:19
  at <anonymous>
```

It seems like the extension was stuck by private Git URLs.

**Comment by pflannery** [comment]
@ajhsu Could provide an example package.json for the problems your getting ?

**Comment by ajhsu** [comment]
Hi @pflannery ,
The following pseudo `package.json` can reproduce the issue on my machine:

```json
{
  "name": "example",
  "version": "1.0.0",
  "dependencies": {
    "example": "git+ssh://git@gitlab.internal.com/web/example.git"
  }
}
```

**Comment by sebastienfi** [comment]
Same here, that's really exciting.

**Comment by pflannery** [comment]
@ajhsu  this is fixed in v0.20.1
